administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -82,16 +82,15 @@ function triggerSave() {
 	<div class="outer-panel" id="right-panel">
 		<div class="inner-panel">
 			<h3><?php echo JText::_('COM_ASSOCIATIONS_ASSOCIATED_ITEM'); ?></h3>
-			<select id="ref-language" name="ref-language">
-				<?php echo JHtml::_('select.options', JHtml::_('contentlanguage.existing', false, true), 'value', 'text'); ?>
-			</select>
+			<?php echo $this->form->getInput('itemlanguage'); ?>
```

Comment:
How do I save this input's state so when I reload the page it's pre-selected with the loaded language?